shared/src/commonTest/kotlin/co/nimblehq/avishek/phong/kmmic/presentation.module/LogInViewModelTest.kt

```diff
@@ -29,8 +29,8 @@ class LogInViewModelTest {
 
     @BeforeTest
     fun setUp() {
-        viewModel = LogInViewModel(logInUseCase)
         Dispatchers.setMain(mainThreadSurrogate)
```

Comment:
Could you please check if we still need to call `Dispatchers.setMain(mainThreadSurrogate)`? If not, we should remove it.

Reply:
This was solved by rebasing